.eslintrc.js

```diff
@@ -5,12 +5,8 @@ module.exports = {
     tsconfigRootDir: __dirname,
     project: ['./tsconfig.json'],
   },
-  ignorePatterns: ["superface/**/*.js", "superface/types", "*.config.js"],
```

Comment:
Why the formatting is changes? It is not happening with `yarn format:fix` and `yarn lint:fix` on shared `chore/update-sf-dependencies`.

Reply:
I'll revert those 3 commits and make it work without these weird changes. 👍🏻